app/models/application_letter.rb

```diff
@@ -46,7 +45,7 @@ def after_deadline?
   # @param none
   # @return [Boolean] true if no status changes are allowed anymore
```

Comment:
Not your code, but a function that checks if a change is allowed returns true when its not?